### ISSUE TITLE
Mozilla plot point size on scattergl plots is not equal as in Chrome and as in SVG plots

### ISSUE THREAD

**Issue by ifll** [plotly/plotly.js#1287]
I have been doing some trials on my own about scatter and scattergl 2D plots. This is what I found:

SVG plot:

- Mozilla performs good and exactly the same as Chrome when inputing a list of sizes for the points being displayed. Length(Points) == Length(List of sizes)

- Chrome performs good an exactly the same as Mozilla in SVG when inputing a list of sizes for the points being displayed. Length(Points) == Length(List of sizes)

WebGL plot:

- Mozilla performs good but different as Chrome in WebGL and SVG and different as Mozilla in SVG as well.

- Chrome performs good and same as Chrome in SVG and Mozilla in SVG but different as Mozilla in WebGL

To summarize, at some point the rendering of the scattergl plot in mozilla oversizes the points even if you input a specific point size. Chrome in SVG and WebGL perform the same

Would that be a bug? Would this be solved?

Thank you very much!!

**Comment by etpinard** [comment]
Can you share a reproducible example?

**Comment by ifll** [comment]
Check out the point size differences between chrome and mozilla running this two codepends:

scatter: https://codepen.io/anon/pen/pRjLwL
scattergl: https://codepen.io/anon/pen/dNYmVP

FYI I was using those browsers on UBUNTU 16.04

Chrome: 55.0.2883.87
Mozilla: 50.1.0

**Comment by ifll** [comment]
In chrome there is a very small difference on point size, but in mozilla there is a huge one with webGL...

**Comment by ifll** [comment]
Between SVG implementations there is no difference between browsers. So Chrome perfomes almost exactly as in SVG, but Mozilla changes a lot... Too much for the app I am doing... and I can't force the useres to use chrome....

**Comment by etpinard** [comment]
Thanks for the report

cc @dfcreative

**Comment by ifll** [comment]
Any idea when this could be solved?

**Comment by rreusser** [comment]
FWIW I can't confirm that I'm seeing this. For the `scattergl` plot in mozilla 50.1.0 in OS X 10.11.4, I see: 

<img width="446" alt="screen shot 2017-01-10 at 11 09 05" src="https://cloud.githubusercontent.com/assets/572717/21813790/4f48ecc8-d725-11e6-8110-44fd3117729a.png">

So there may be a component of OS or hardware dependency involved.

**Comment by etpinard** [comment]
Thanks for checking @rreusser !

**Comment by ifll** [comment]
Mozilla 50.1.0 Ubuntu 16.04 SVG plot

![captura de pantalla de 2017-01-10 17-20-27](https://cloud.githubusercontent.com/assets/16896039/21814383/6bd79964-d759-11e6-90a4-96bd0fee68f7.png)

Mozilla 50.1.0 Ubuntu 16.04 WebGL plot

![captura de pantalla de 2017-01-10 17-20-42](https://cloud.githubusercontent.com/assets/16896039/21814384/6be16b4c-d759-11e6-9e5c-a5927c9dba29.png)

**Comment by etpinard** [comment]
@ifll yes, I can confirm the behavior in Ubuntu 16.04

**Comment by ifll** [comment]
It seems to be Ubuntu Specific, as I have checked in windows and it worked...

**Comment by dfcreative** [comment]
We use monospace default system font for rendering symbols, it is likely that mozilla has whether different monospace font or uses different rendering approach.
@ifll can you show screenshots of chrome/mozilla webgl plot?

**Comment by ifll** [comment]
Currently I cannot dedicate some time on it, I need to face other stuff. Maybe during the week or on the next week I can find some time and try to provide them. Sorry for this and hope it is not an inconvinience....

**Comment by dfcreative** [comment]
https://github.com/plotly/plotly.js/commit/1e300c4a25c8c9e4be50bc64b2ca807e922fef2d adds sizes test

**Comment by etpinard** [comment]
done in #2258
